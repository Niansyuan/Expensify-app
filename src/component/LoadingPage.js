import React from "react";

export default () => {
    return (
        <div className="loader">
            <img className="loader__image" src='/image/Loading.gif' />
            <p className="loader__title">LOADING ...</p>
        </div>
    );
};