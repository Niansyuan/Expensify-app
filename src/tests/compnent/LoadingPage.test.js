import React from "react";
import { shallow } from "enzyme";
import LoadingPage from "../../component/LoadingPage";

test('should render LoadingPage correctly', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
});