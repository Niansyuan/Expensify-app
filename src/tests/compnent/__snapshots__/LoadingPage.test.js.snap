// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/image/Loading.gif"
  />
  <p
    className="loader__title"
  >
    LOADING ...
  </p>
</div>
`;
